src/distance/NominalConfigurationRanker.cpp

```diff
@@ -0,0 +1,36 @@
+#include "aikido/distance/NominalConfigurationRanker.hpp"
+
+namespace aikido {
+namespace distance {
+
+using statespace::dart::ConstMetaSkeletonStateSpacePtr;
+using statespace::CartesianProduct;
+using ::dart::dynamics::ConstMetaSkeletonPtr;
+
+//==============================================================================
+NominalConfigurationRanker::NominalConfigurationRanker(
+    ConstMetaSkeletonStateSpacePtr metaSkeletonStateSpace,
+    ConstMetaSkeletonPtr metaSkeleton,
+    const std::vector<statespace::StateSpace::State*> ikSolutions)
+  : ConfigurationRanker(metaSkeletonStateSpace, metaSkeleton, ikSolutions)
+{
+  // Do nothing
+}
+
+//==============================================================================
+double NominalConfigurationRanker::evaluateIKSolution(
+    statespace::StateSpace::State* solution) const
+{
+  auto currentState = mMetaSkeletonStateSpace->createState();
```

Comment:
Should this just be `mMetaSkeletonStateSpace->getScopedStateFromMetaSkeleton(mMetaSkeleton.get())`?